pkg/operator/ceph/object/objectstore.go

```diff
@@ -392,55 +392,49 @@ func getZoneEndpoints(objContext *Context, serviceEndpoint string) ([]string, bo
 	return zoneEndpointsList, isEndpointAlreadyExists, nil
 }
 
+func createMultisiteConfigurations(objContext *Context, expectJSON bool, args ...string) error {
```

Comment:
1. If certain args are required, name them in the func signature instead of extracting them from `args`.
2. `expectJSON` doesn't look needed
```suggestion
func createMultisiteConfigurations(objContext *Context, configType, configTypeArg string, args ...string) error {
```

Reply:
Yaa it looks good, but I have to pass the same value twice, lemme know if it looks good
OR
I have to append it in the arg later

Reply:
PS: But it is a small list